doc/news/_preparation_next_release.md

```diff
@@ -41,7 +41,42 @@ From now on `keyNew ("/array/#10", KEY_END)` will create a `Key` with name `/arr
 The whole internal implementation for `keySetName`, `keyAddName`, etc. has been completely rewritten.
```

Comment:
The text above somehow implies that there are only two big changes. Put everything in an itemize, like done later.